### PR TITLE
Fp 1815 exception page 1/23

### PR DESCRIPTION
**Pull request by sophia-massie**
## Overview

Updated Threshold Exception page after moving to staging and testing.

## Related

- [FP-1815](https://jira.tacc.utexas.edu/browse/FP-1815)
- requires https://github.com/TACC/Core-CMS-Custom/pull/72 Follow on PR from this previous PR

## Changes

Updated threshold % request and field code input audit. Removed RT credential check in exception/views causing 500 error. Need to debug.

## Testing

1. Test[ both the threshold form and other form pages](http://localhost:8000/submissions/exception/) with the following data and selections
2. Change line 30 and 93 in exception/views.py to the following snippet. 
3. In business name drop down under Acknowledgment of Terms, select chcd ID - 1
<summary>Snippet</summary><details> [(1, 'TESTGOLD', 10000000, 'thbrown', 'chcd'), (4, 'TESTMRTN', 10000003, 'thbrown', 'chcd')] </details>

## UI
Threshold Page 1/2
Screen Shot 2023-02-06 at 5 12 10 PM

Threshold page 2/2
Screen Shot 2023-02-06 at 5 14 24 PM

Other exception page 1/2
Screen Shot 2023-02-06 at 5 15 01 PM

Other exception page 2/2
Screen Shot 2023-02-06 at 5 15 16 PM

### CODE REVIEW

File: apcd-cms/src/apps/exception/views.py [hunk 1]
```diff
@@ -1,175 +1,151 @@
...
-        context = super(ExceptionThresholdFormView, self).get_context_data(
-            *args, **kwargs
-        )
-        submitter_cont =  apcd_database.get_submitter_for_exception(
-            request.user.username
-        )
+    def get_context_data(self, *args, **kwargs):
+        context = super(ExceptionThresholdFormView, self).get_context_data(*args, **kwargs)
+
+        submitters = [(1, 'TESTGOLD', 10000000, 'thbrown', 'chcd'), (4, 'TESTMRTN', 10000003, 'thbrown', 'chcd')]
```
Comment (fnets) [inline apcd-cms/src/apps/exception/views.py:30]:
```suggestion
```

File: apcd-cms/src/apps/exception/views.py [hunk 2]
```diff
@@ -1,175 +1,151 @@
...
+        return super(ExceptionOtherFormView, self).dispatch(request, *args, **kwargs)
 
-    def get_context_data(self, request, *args, **kwargs):
+    def get_context_data(self, *args, **kwargs):
         context = super(ExceptionOtherFormView, self).get_context_data(*args, **kwargs)
-        submitter_cont =  apcd_database.get_submitter_for_exception(
-            request.user.username
-        )
+
+        submitters = [(1, 'TESTGOLD', 10000000, 'thbrown', 'chcd'), (4, 'TESTMRTN', 10000003, 'thbrown', 'chcd')]
```
Comment (fnets) [inline apcd-cms/src/apps/exception/views.py:93]:
```suggestion
```

**Review by fnets — CHANGES_REQUESTED** [review]
A couple of fake data assignments made it into the PR. Mind taking those out?

**Review by sophia-massie — COMMENTED** [review]
Changes updated

**Review by fnets — APPROVED** [review]
LGTM and really readable! Good work

File: apcd-cms/src/apps/exception/templates/exception_submission_form/exception_submission_error.html [hunk 3]
```diff
@@ -12,7 +12,7 @@ <h1>Request to Submit Exception</h1>
     <p class="c-message c-message--error">
         An error occurred during your exception request submission. For help, <a href="/workbench/dashboard">submit a ticket</a>.
     </p>
-    <a class="c-button c-button--primary" href="/submissions/exceptions/">Go to Exception Forms</a>
+    <a class="c-button c-button--primary" href="/submissions/exceptions/">Go Back to Exception Form</a>
```
Comment (edmondsgarrett) [inline apcd-cms/src/apps/exception/templates/exception_submission_form/exception_submission_error.html:15]:
```suggestion
    <a class="c-button c-button--primary" href="/submissions/exception/">Go Back to Exception Form</a>
```
Back button is broken

File: apcd-cms/src/apps/exception/templates/exception_submission_form/exception_form_success.html [hunk 4]
```diff
@@ -8,6 +8,6 @@ <h1>Request to Submit Exception</h1>
     <p>
         Your exception submission was successful.
     </p>
-    <a class="c-button c-button--primary" href="/submissions/exception">Go to Exception Forms</a>
+    <a class="c-button c-button--primary" href="/submissions/exception">Go Back to Exception Form</a>
```
Comment (edmondsgarrett) [inline apcd-cms/src/apps/exception/templates/exception_submission_form/exception_form_success.html:11]:
```suggestion
    <a class="c-button c-button--primary" href="/submissions/exception/">Go Back to Exception Form</a>
```
Back button consistency

**Review by edmondsgarrett — APPROVED** [review]
LGTM!

**Pull request merged**